assets/blueprints/blueprint.json

```diff
@@ -0,0 +1,21 @@
+{
+	"$schema": "https://playground.wordpress.net/blueprint-schema.json",
+	"landingPage": "/wp-admin/site-editor.php",
+	"plugins": [ "gutenberg" ],
+	"login": true,
+	"features": {
+		"networking": true
+	},
+	"preferredVersions": {
+		"php": "8.0",
+		"wp": "latest"
+	},
+	"steps": [
+		{
+			"step": "setSiteOptions",
+			"options": {
+				"blogname": "Testing Gutenberg"
+			}
+		}
```

Comment:
When loading this Blueprint, I am seeing a "sorry, not found" kind of message. Here is a screenshot with the message highlighted in red:
<img src="https://github.com/user-attachments/assets/be5a4ec1-d7ca-48b3-afa3-e3c5506472d7" width="500">

Would it be good for us to add a page to avoid the message? Maybe it is good to just see the default view for a new site. I'm not sure.

If desired, we could add the following step to create a page that avoids the "sorry" message.

```suggestion
		},
		{
			"step": "runPHP",
			"code": "<?php require_once 'wordpress/wp-load.php';\n$page_args = array(\n'post_type'    => 'page',\n'post_status'  => 'publish',\n'post_title'   => 'Gutenberg Live Preview',\n'post_content' => '<!-- wp:paragraph --><p>This is a sample page created for the Gutenberg Live Preview.</p><!-- /wp:paragraph -->',\n);\n$page_id = wp_insert_post( $page_args );update_option( 'page_on_front', $page_id );update_option( 'show_on_front', 'page' );"
		}
```

If that step is included the site editor content looks like this:
<img src="https://github.com/user-attachments/assets/82f40c26-15d3-4700-811b-3f369ef3dcbd" width="500">

Reply:
I can see how that's confusing, yes. And it's the putting  What you see on the Blog home page that includes the **No Results** block of the template
![Screenshot 2024-12-11 at 12 06 44](https://github.com/user-attachments/assets/8e9dfd36-79a2-4039-91d8-788588d5a810)

I will think about it. We might not need an additional page if we land on the Sample Page instead of the Blog Home page.